addons/nvidia/nvidia.yaml

```
@@ -8,9 +7,9 @@ metadata:
     kubeaddons.mesosphere.io/name: nvidia
     kubeaddons.mesosphere.io/provides: nvidia
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-8"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-9"
```

Increment the addon patch version to account for the patch version change of the chart. Only increment release version when the only change is to the addon resource alone.

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.1-0"
```